sql/catalyst/src/main/scala/org/apache/spark/sql/types/Decimal.scala

```diff
@@ -184,68 +184,56 @@ final class Decimal extends Ordered[Decimal] with Serializable {
     this
   }
 
-  def toBigDecimal: BigDecimal = {
-    if (decimalVal.ne(null)) {
-      decimalVal
-    } else {
-      BigDecimal(longVal, _scale)
-    }
+  def toBigDecimal: BigDecimal = if (decimalVal.ne(null)) {
```

Comment:
can we avoid pure code style change? the previous code style was not wrong.

Reply:
OK